src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -343,6 +343,17 @@ QVector<TrackerEntry> TorrentHandleImpl::trackers() const
     return entries;
 }
 
+bool TorrentHandleImpl::tracker(const QString &trackerUrl, TrackerEntry &entry) const
+{
+    for (const auto &tracker : m_nativeHandle.trackers()) {
+        if (QString::fromStdString(tracker.url) == trackerUrl) {
```

Comment:
It's not optimal to convert each `tracker.url`. I would convert `trackerUrl` to `std::string` once and then compare with it. Seems you can even use some standard algorithm in that case.